src/main.rs

```diff
@@ -407,6 +409,38 @@ fn main() {
         process::exit(0);
     }
 
+    // Language list according to
+    // https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#language
+    let languages = if let Some(ref lang) = args.flag_language {
+        // Language overwritten by console argument
+        vec![lang.clone()]
+    } else if let Ok(lang) = std::env::var("LANG") {
+        let locales = std::env::var("LANGUAGE").unwrap_or_default();
+        let mut locales: Vec<&str> = locales.split(':').collect();
+        // According to the client specification, these should be appended
+        locales.push(&lang[..]);
+        locales.push("en");
+
+        let mut lang_list = Vec::new();
+        for locale in locales.iter() {
+            if locale.len() >= 5 && locale.chars().nth(2) == Some('_') {
+                // Language with county code
+                lang_list.push(&locale[..5]);
+            }
+            if locale.len() >= 2 {
+                // Language code
+                lang_list.push(&locale[..2]);
+            }
+        }
+
+        lang_list.dedup();
```

Comment:
Note: This will only deduplicate consecutive repeated values, so `LANG=es LANGUAGE=fr,es,de` would result in `fr,es,de,es`.